app/css/main.css

```diff
@@ -353,38 +348,74 @@ section p {
 article {
   flex: 1;
   margin: 40px 0;
-  /* Set min-width so article width will shrink
-  (and code snippets will horizontally scroll). */
   min-width: 1px;
+  max-width: 720px;
```

Comment:
@yuinchien This broke the centered styling of blog post content. This hasn't been deployed yet, but can be seen locally:

![image](https://user-images.githubusercontent.com/1319720/50523301-37d24a00-0a84-11e9-91d6-54d73d083801.png)